modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordBinaryManager.kt

```diff
@@ -278,7 +274,18 @@ class MirrordBinaryManager {
             }
 
             val binary = MirrordBinary(output)
-            if (requiredVersion == null || requiredVersion == binary.version) {
+
+            val isRequiredVersion = try {
```

Comment:
This check is modified only here, what about `findBinaryInStorage`?

Reply:
for the CI we only care about the path as we add the latest build to the path